api/v1alpha1/flowcollector_types.go

```diff
@@ -30,10 +30,6 @@ const (
 	DeploymentModelKafka  = "KAFKA"
 )
 
-func (spec *FlowCollectorSpec) UseEBPF() bool  { return spec.Agent.Type == AgentEBPF }
-func (spec *FlowCollectorSpec) UseIPFIX() bool { return spec.Agent.Type == AgentIPFIX }
-func (spec *FlowCollectorSpec) UseKafka() bool { return spec.DeploymentModel == DeploymentModelKafka }
-
```

Comment:
(by the way, I don't think it was necessary to add the new helper to the v1alpha1 version, and maybe not doing so would have spared us from the controversy 😅)

Reply:
Operator controller now point to v1beta1, this functions were not used anyway